tracer/test/Datadog.Trace.TestHelpers/ProfilerHelper.cs

```diff
@@ -68,19 +68,22 @@ public class ProfilerHelper
             startInfo.RedirectStandardOutput = true;
             startInfo.RedirectStandardError = true;
             startInfo.RedirectStandardInput = redirectStandardInput;
+
             if (!string.IsNullOrEmpty(workingDirectory))
             {
                 startInfo.WorkingDirectory = workingDirectory;
             }
 
-            var process = Process.Start(startInfo);
-
-            if (process != null)
+            if (EnvironmentTools.IsWindows())
             {
-                MemoryDumpHelper.MonitorCrashes(process.Id);
+                using var suspendedProcess = NativeProcess.CreateProcess.StartSuspendedProcess(startInfo);
```

Comment:
Is it worth having a toggle switch to enable/disable this?
(I expect the answer to be "what's the point?")

Reply:
Let's just add it when/if we need it 😛